co-noir/co-acvm/src/solver.rs

```diff
@@ -212,6 +222,43 @@ impl<N: Rep3Network> Rep3CoSolver<ark_bn254::Fr, N> {
     }
 }
 
+impl<N: ShamirNetwork> ShamirCoSolver<ark_bn254::Fr, N> {
+    pub fn from_network<P>(
+        network: N,
+        threshold: usize,
+        compiled_program: ProgramArtifact,
+        prover_path: P,
+    ) -> eyre::Result<Self>
+    where
+        PathBuf: From<P>,
+    {
+        // TODO we are not creating any randomness here
```

Comment:
do we need to?

Reply:
Not yet, but may be required in the future. Just kept the note here to not forget